Notebooks/QC/contamination.ipynb

```diff
@@ -91,12 +85,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "kraken-report --db ./minikraken_20171013_4GB --print_header \\\n",
```

Comment:
That looks very much like what is created in the pipelines! Worth investigating if this is an old option that got deprecated.